examples/nlp/text_classification/conf/config.yaml

```diff
@@ -0,0 +1,81 @@
+# Sentence classification with pretrained BERT models
+
+io:
+  work_dir: ./outputs
+  checkpoint_dir: null
+  save_epoch_freq: 1
+  save_step_freq: -1
+
+pl:
+  trainer:
+    gpus: 1 # the number of gpus, 0 for CPU
+    num_nodes: 1
+    max_epochs: 10
+    max_steps: null # precedence over max_epochs
+    accumulate_grad_batches: 1 # accumulates grads every k batches
+    amp_level: O0 # O1/O2 for mixed precision
+    distributed_backend: ddp
+
+model:
+  data_dir: ??? # /path/to/data
+  class_balancing: null # or weighted_loss
+
+  train_ds:
+    prefix: train
+    batch_size: 32
+    shuffle: true
+    num_samples: -1
+    num_workers: 2
+    pin_memory: false
+    use_cache: true
+    drop_last: true
+
+  validation_ds:
```

Comment:
Why data loaders configs are part of the model?

Reply:
ModelPT = code for model to be created with weights, it's training config, it's data preprocessors and post processors, it's training and eval loop, and finally it's data loaders. That's why all of this will be rolled into one.

Reply:
Data is part of NeMo Model definition.

Reply:
think of Models as pretty much everything math-related to the experiment  minus engineering details (GPUs, dist backend).

Reply:
Optimizers and schedulers are not part of the model?